examples/material_gallery/test_driver/scroll_perf_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('scrolling performance test', () {
+    FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+    });
+
+    tearDownAll(() async {
+      if (driver != null)
+        driver.close();
+    });
+
+    test('measure', () async {
+      Timeline timeline = await driver.traceAction(() async {
+        // Find the scrollable stock list
+        ObjectRef stockList = await driver.findByValueKey('Gallery List');
+        expect(stockList, isNotNull);
+
+        await driver.tap(await driver.findByText('Demos'));
+        await driver.tap(await driver.findByText('Components'));
+        await driver.tap(await driver.findByText('Style'));
+
+        // Scroll down
+        for (int i = 0; i < 5; i++) {
+          await driver.scroll(stockList, 0.0, -300.0, new Duration(milliseconds: 300));
+          await new Future<Null>.delayed(new Duration(milliseconds: 500));
```

Comment:
Not sure what we're waiting for here, not sure what the wait's significance is vis performance, not clear what 500ms is based on.